app/views/accessibilityStatement.scala.html

```diff
@@ -0,0 +1,236 @@
+@(isLoggedIn: Boolean, name: String, isJudgmentUser: Boolean)(implicit messages: Messages, request: RequestHeader)
+@defining("Accessibility statement") { title =>
+  @main(title, isLoggedIn = isLoggedIn, name = name, isJudgmentUser = isJudgmentUser) {
+    <h1 class="govuk-heading-l">@title</h1>
+    <p class="govuk-body"> This statement applies to content published on Transfer Digital Records (TDR) only.</p>
+    <p class="govuk-body">
+      TDR is run by the Digital Archiving department at The National Archives.
+      It is designed to be used by as many people as possible.
+      The text should be clear and simple to understand. You should be able to:
+    </p>
+    <ul class="govuk-list govuk-list--bullet govuk-list--spaced">
+      <li>zoom in up to 300% without problems</li>
+      <li>navigate most of the website using just a keyboard</li>
+      <li>navigate most of the website using speech recognition software</li>
+      <li>
+        use most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver</li>
+    </ul>
+    <p class="govuk-body">The website text has been made as simple to understand as possible and tested with real users.</p>
+    <h2 class="govuk-heading-m">How accessible this website is</h2>
+    <p class="govuk-body">
+      In its current iteration, the vast majority of TDR fully meets Web Content Accessibility Guidelines (WCAG) 2.1 AA standards.</p>
+    <p class="govuk-body">Parts of this website that are not fully accessible include:</p>
+    <ul class="govuk-list govuk-list--bullet govuk-list--spaced">
+      <li>some heading elements are not consistent</li>
+      <li>some images do not have image descriptions</li>
+      <li>some buttons are not correctly identified</li>
+      <li>some error messages are not sufficient to identify or rectify errors</li>
+      <li>some custom elements are inaccessible for assistive technologies</li>
+    </ul>
+
+
+    <p class="govuk-body">
+      As part of ongoing development, design iterations incorporate a progressive enhancement approach
+      and TDR is continuously tested to ensure that it meets WCAG 2.1 AA standards wherever possible.
+    </p>
+    <h2 class="govuk-heading-m">Feedback and contact information</h2>
+    <p class="govuk-body">If you need information on this website in a different format:</p>
+    <p class="govuk-body">Email: <a href="mailto:tdr@@nationalarchives.gov.uk">tdr@@nationalarchives.gov.uk</a></p>
```

Comment:
Could you use the `Messages` for the email address to keep this consistent with the other pages: `nationalArchives.email` 

Also mean updating is easier should the email address change in the future

Reply:
Good spot, I've swapped it out for the `@Messages` one.